templates/observability/what-is-observability.html

```diff
@@ -0,0 +1,273 @@
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_4' -%}
+        <h3 class="p-heading--5">Production profiling</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_4' -%}
+        <p>
+          Production profiling identifies hot spots of resource consumption by continuously sampling, with limited performance overhead, which parts of your application consume CPU cycles, memory and, depending on the runtime, the amount of wait time, or asynchronous wait. Production profiling is usually very useful in getting you started when troubleshooting latency and memory usage spikes in the absence of more detailed telemetry. The "production" reference is due to the tradeoffs in the design of the profiler to optimize for minimal overhead, as opposed to profilers used in development that tend to optimize for precision.
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_5' -%}
+        <h3 class="p-heading--5">Real user monitoring</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_5' -%}
+        <p>
+          Real user monitoring is about collecting telemetry about user sessions occurring via interfaces like web pages, mobile applications and sometimes IoT devices; real user monitoring is closely related with distributed tracing, in which a user session is correlated with one or more distributed traces, each describing, for example, how one asset or XHR request is served to the user interface.
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_6' -%}
+        <h3 class="p-heading--5">Synthetic monitoring</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_6' -%}
+        <p>
+          Synthetic monitoring checks are continuous test systems, mostly production ones, to ensure that particular APIs or even entire user flows in UIs, like a checkout process, work as intended; in other words, synthetic monitoring checks are smoke tests for APIs and UIs.
+        </p>
+      {%- endif -%}
+    {%- endcall -%}
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Telemetry correlation for deeper insights</h2>
+      </div>
+      <div class="col">
+        <p class="u-sv4">
+          <b>Making the most out of your metrics, logs, distributed traces and more</b>
+          <br />
+          The various telemetry types provide you with more insight when they are correlated with one another.
+        </p>
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/2e3a4a28-telemetry-correlation.png",
```

Comment:
According to the doc, this diagram should've been replaced. Left a [q](https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit?disco=AAABZRXtYcw)